core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -73,6 +78,7 @@ private[op] trait TransmogrifierDefaults {
   val TrackInvalid: Boolean = false
   val TrackTextLen: Boolean = false
   val MinDocFrequency: Int = 0
+  val MaxPercentCardinality = 1.0
```

Comment:
let's keep using the same const `val MaxPercentCardinality = OpOneHotVectorizer. MaxPercentCardinality`